src/components/Listing.tsx

```diff
@@ -1,29 +1,260 @@
+        background: `linear-gradient(90deg, rgba(255,255,255,0.25) 0%, ${COLORS[8]} 10%)`,
+      },
+      "& .MuiButtonBase-root.chip-10": {
+        background: `linear-gradient(90deg, rgba(255,255,255,0.5) 25%, ${COLORS[9]} 100%)`,
+      },
+      "& .MuiButtonBase-root.chip-10:hover": {
+        background: `linear-gradient(90deg, rgba(255,255,255,0.25) 0%, ${COLORS[9]} 10%)`,
+      },
+      "& .MuiButtonBase-root.chip-11": {
+        background: `linear-gradient(90deg, rgba(255,255,255,0.5) 25%, ${COLORS[10]} 100%)`,
+      },
+      "& .MuiButtonBase-root.chip-11:hover": {
+        background: `linear-gradient(90deg, rgba(255,255,255,0.25) 0%, ${COLORS[10]} 10%)`,
+      },
+      "& .MuiButtonBase-root.chip-12": {
+        background: `linear-gradient(90deg, rgba(255,255,255,0.5) 25%, ${COLORS[11]} 100%)`,
+      },
+      "& .MuiButtonBase-root.chip-12:hover": {
+        background: `linear-gradient(90deg, rgba(255,255,255,0.25) 0%, ${COLORS[11]} 10%)`,
+      },
+      "& .MuiButtonBase-root.chip-13": {
+        background: `linear-gradient(90deg, rgba(255,255,255,0.5) 25%, ${COLORS[12]} 100%)`,
+      },
+      "& .MuiButtonBase-root.chip-13:hover": {
+        background: `linear-gradient(90deg, rgba(255,255,255,0.25) 0%, ${COLORS[12]} 10%)`,
+      },
+      "& .MuiButtonBase-root.chip-14": {
+        background: `linear-gradient(90deg, rgba(255,255,255,0.5) 25%, ${COLORS[13]} 100%)`,
+      },
+      "& .MuiButtonBase-root.chip-14:hover": {
+        background: `linear-gradient(90deg, rgba(255,255,255,0.25) 0%, ${COLORS[13]} 10%)`,
+      },
+    },
+  },
+  tabs: {
+    alignItems: "flex-end",
+    borderRight: `1px solid rgba(240,240,240,1)`,
+    inkBarStyle: { background: "#000", height: "5px", marginTop: "-5px" },
+  },
+  tab: {
+    opacity: 1,
+    background: `linear-gradient(90deg, rgba(255,255,255,0.5) 25%, ${COLORS[14]} 100%)`,
+    width: 200,
+    alignContent: "flex-end",
+    alignSelf: "flex-end",
+    justifyItems: "flex-end",
+    justifySelf: "flex-end",
+    justifyContent: "flex-end",
+    textAlign: "right",
+    textTransform: "none",
+    "&:focus": {
+      background: `linear-gradient(90deg, rgba(255,255,255,0) 0%, ${COLORS[14]} 10%)`,
     },
-    {
-      title: "Previously read links",
-      items: [...items.filter((item) => item.alreadyRead)],
-      visible: false,
+    "&:hover": {
+      background: `linear-gradient(90deg, rgba(255,255,255,0.25) 0%, ${COLORS[14]} 10%)`,
+      opacity: 1,
     },
-  ];
-  const resultingTabs = [];
-  for (const tab of tabs) {
-    if (tab.visible || (tab.items.length && !tab.visible))
-      resultingTabs.push(
-        <div key={tab.title}>
-          <h2>
-            {tab.title} ({tab.items.length})
-          </h2>
+    // "&$selected": {
+    //   opacity: 0,
+    // },
+    "& span.MuiTab-wrapper, & span.MuiTouchRipple-root": {
+      alignItems: "flex-end",
+    },
+  },
+}));
+
+type ItemsToTabConnectorType = {
```

Comment:
I am quite unsatisfied with this solution but could not figure anything better at the moment. Could we have a look at this next time @erikvanek?

Reply:
We should I guess. Already the type name induces some code smells somewhere...